### PR TITLE
Fix directory permission issue in linux containers

### PR DESCRIPTION
**Pull request by akoken**
Fixes #325

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,27 +30,19 @@ RUN dotnet publish -a $TARGETARCH -c Release -o /app --no-restore --self-contain
 # Final stage/image
 FROM mcr.microsoft.com/dotnet/runtime:8.0 AS runtime
 
-# Determine the distribution and install libaio1
```
Comment (badrishc) [inline Dockerfile:33]: why was this part removed in the PR?

Comment (akoken) [reply Dockerfile:33]: The default .NET 8 base image is [Debian](https://github.com/dotnet/dotnet-docker/discussions/5709), and this will remain the case at least until .NET 10. There is no reason to keep that `if statements` in Dockerfile.

**Review by badrishc — APPROVED** [review]
